code/style.css

```diff
@@ -1,58 +1,147 @@
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: 'Montserrat', sans-serif;
+  background-color: rgb(27, 27, 27);
+}
+
+.start-text {
+  font-weight: 900;
+  font-size: 100px;
+  text-align: center;
+  padding: 30px;
+  margin-top: 30px;
+  margin-bottom:  -100px;
+  color: #ffffff;
 }
 
 p {
-  margin: 1px;
-  font-weight: 300;
-  background-color: #fff;
+  margin: 10px;
+  font-weight: 500;
+  font-size: 18px;
+  color: #fff;
+}
+
+.killer-deck h2 {
+  color: #527262;
+}
+
+.weapon-deck h2 {
+  color: #C46049;
 }
 
+.room-deck h2 {
+  color: #C7A54B;
+}
 .cards {
   margin: 50px auto;
   width: 90%;
   height: 600px;
-  background: #ccc;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  border-radius: 30px;
+  max-width: 1200px;
+
+}
+
+.killer-info img {
+  border-radius: 10%;
 }
 
 .killer-deck {
-  width: 250px;
+  width: 300px;
   height: 400px;
-  background: #000;
+  background: #A6C8A6;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  margin: 15px;
+  cursor: pointer;
 }
 
 .weapon-deck {
-  width: 250px;
+  width: 300px;
   height: 400px;
-  background: rgb(10, 53, 10);
+  background: #EB8576;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  margin: 15px;
+  cursor: pointer;
 }
 
 .room-deck {
-  width: 250px;
+  width: 300px;
   height: 400px;
-  background: midnightblue;
+  background: #EAC76B;
 
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  margin: 15px;
+  cursor: pointer;
 }
```

Comment:
Nice with different colors on the decks, makes it easy to spot the difference between them :)